file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,278 @@
+      "! <p class="shorttext">Turnkey Implementation Type</p>
+      "! Type of turnkey implementation
+      turnkey_type               TYPE c LENGTH 10,
+      "! <p class="shorttext">Scenario Technology</p>
+      "! Scenario technology
+      scenario_technology        TYPE c LENGTH 20,
+      "! <p class="shorttext">Prediction Class</p>
+      "! Prediction class
+      prediction_class           TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Prerequisite Check Class</p>
+      "! Prerequisite check class
+      prerequisite_check_class   TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Scenario Technical ID</p>
+      "! Scenario technical ID
+      scenario_technical_id      TYPE c LENGTH 64,
+      "! <p class="shorttext">State</p>
+      "! State of an intelligent scenario
+      state                      TYPE c LENGTH 10,
+      "! <p class="shorttext">ISLM Framework Version</p>
+      "! ISLM framework version
+      islm_version               TYPE p LENGTH 4 DECIMALS 2,
+      "! <p class="shorttext">Turnkey Class</p>
+      "! Turnkey class
+      turnkey_class              TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Usage Type</p>
+      "! Usage type of an intelligent scenario
+      usage_type                 TYPE c LENGTH 10,
+      "! <p class="shorttext">OAuth Profile</p>
+      "! OAuth 2.0 client profile
+      oauth_profile              TYPE c LENGTH 30,
+    END OF ty_intelligent_scenario,
+
+    "! <p class="shorttext">Dataset Binding</p>
+    "! Scenario dataset binding
+    BEGIN OF ty_binding,
+      "! <p class="shorttext">Binding Name</p>
+      "! Name of the binding
+      name      TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Reference object</p>
+      "! Reference object(for example - CDS view)
+      reference TYPE c LENGTH 36,
+    END OF ty_binding,
+
+    "! <p class="shorttext">Signature</p>
+    "! Scenario signature
+    BEGIN OF ty_signature,
+      "! <p class="shorttext">Signature Name</p>
+      "! Signature name
+      input_name           TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Signature Type</p>
+      "! Signature type
+      signature_type       TYPE c LENGTH 32,
+      "! <p class="shorttext">Description</p>
+      "! Description
+      description          TYPE c LENGTH 100,
+      "! <p class="shorttext">Field Name</p>
+      "! Table or view field name
+      field_name           TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Position</p>
+      "! Position of the field
+      field_position       TYPE i,
+      "! <p class="shorttext">Storage Type</p>
+      "! Storage type of the field
+      storage_type         TYPE c LENGTH 100,
+      "! <p class="shorttext">Role of Field</p>
+      "! Role of field in an intelligent scenario (such as key, target)
+      field_type           TYPE c LENGTH 30,
```

Comment:
What is the better name for this field "Role of Field" or "Field Type". The title and the abap field name should be consistent.